woocommerce/woocommerce-gutenberg-products-block
# Add global style for Active Filters block

This is a part of #4965.

This PR:

- Adds text color support



![image](https://user-images.githubusercontent.com/4463174/147587196-86b1ee61-8ec9-41c3-a9dd-0a709b68e513.png)
*editor side*



![image](https://user-images.githubusercontent.com/4463174/147587235-a88b6b93-d8ee-4f27-8c1e-d9172877a41a.png)
*frontend side*

### Testing


### Manual Testing

How to test the changes in this PR:

Check out this branch:

1. Install and enable the `Gutenberg` plugin.
2. Install and enable the `TT1 Blocks` theme.
3. Add the Active Product Filters block block to a post. (you need to add a block like `Filters Product By Attribute`)
4. Go to Color section & verify you can change the text color.
5. Save your changes.
6. Go on the page and check if there are changes.
7. Reset to default using the `Clear` button from Styles > Text color window.
8. Go to Dashboard and select Appearance > Editor (beta).
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the Active Product Filters block is shown Under the `Blocks` section .
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the text color.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.




### Changelog

> Added global styles (text color) to the Active Filters block.
